code/game/objects/items/weapons/pneumaticCannon.dm

```diff
@@ -26,6 +26,7 @@
 	var/fire_mode = PCANNON_FIREALL
 	var/automatic = FALSE
 	var/clumsyCheck = TRUE
+	var/trigger_guard = TRIGGER_GUARD_NORMAL
```

Comment:
?
can_trigger_gun calls can_use_guns, which checks if the item has a trigger_guard (and then checks the user for blah blah ect.)

It's currently working on live because #24383 copypasta'd the HULK & NOGUNS checks from can_use_guns with the trigger_guard check removed.

or am I missing something else that you're trying to point out?